tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -59,7 +61,7 @@ sub reboot_and_wait_up {
         else {
             #login
             #The timeout can't be too small since autoyast installation
-            assert_screen "text-login", 600;
+            assert_screen([qw(text-login sol-fresh-text-login)], 60);
```

Comment:
Is the `sol-fresh-text-login` needle new? Why?

Reply:
Yes, it is a new needle. because the screen of sol console were cleared after system reboot in the SUT.
http://10.67.129.51/tests/2503#step/reboot_and_wait_up_normal/5

I have already set IPMI_SOL_PERSISTENT_CONSOLE=1.

Reply:
IMHO, this may need a new needle, rather than a new needle tag. It is still text-login, similar with sle12/15/tw share the same needle tag, but may need different needles.

Reply:
a little confused. Let me have a try with needle instead of tag.

Reply:
Simply a needle works in both SLE tests in OSD & TW in my own openqa server.

Reply:
Thank you